apps/web/src/components/Profile/components/PersonalInformation/Display.tsx

```diff
@@ -36,128 +53,197 @@ const Display = ({
 }: DisplayProps) => {
   const intl = useIntl();
   const notProvided = intl.formatMessage(commonMessages.notProvided);
+  const { userAuthInfo } = useAuthorization();
+  const navigate = useNavigate();
+  const routes = useRoutes();
+  const applicationContext = useApplicationContext();
 
-  return (
+  // only show email verification if NOT in an application currently
+  const showEmailVerification = !applicationContext.currentStepOrdinal;
+
```

Comment:
Confused, the issue requests that it **_IS_** present in that context

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/f0f75a60-7e9d-417e-a29d-ab121d05aaf8)

Reply:
Registration does not appear to connect there either

Reply:
@gobyrne could you please confirm the scope of this issue?  Just profile or also application and create-account?

Reply:
I figured the other two were here:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/5c405f4e-f285-4f80-bd51-8e7cd5f97470)

Reply:
Addressed below